deploy/test-environments/fleet_api/src/install_cnvm_integration.py

```diff
@@ -0,0 +1,106 @@
+from api.common_api import (
+    get_enrollment_token,
+    get_fleet_server_host,
+    get_build_info,
+)
+from loguru import logger
+from utils import (
+    read_json,
+    save_state,
+)
+
+CNVM_AGENT_POLICY = "../../../cloud/data/agent_policy_cnvm_aws.json"
+CNVM_PACKAGE_POLICY = "../../../cloud/data/package_policy_cnvm_aws.json"
+CNVM_CLOUDFORMATION_CONFIG = "../../../cloudformation/config.json"
+CNVM_ARTIFACT_SUFFIX = "downloads/beats/elastic-agent"
+
+cnvm_agent_policy_data = Path(__file__).parent / CNVM_AGENT_POLICY
+cnvm_pkg_policy_data = Path(__file__).parent / CNVM_PACKAGE_POLICY
+cnvm_cloudformation_config = Path(__file__).parent / CNVM_CLOUDFORMATION_CONFIG
+
+
+def load_data() -> Tuple[Dict, Dict]:
+    """Loads data.
+
+    Returns:
+        Tuple[Dict, Dict]: A tuple containing the loaded agent and package policies.
+    """
+    logger.info("Loading agent and package policies")
+    agent_policy = read_json(json_path=cnvm_agent_policy_data)
+    package_policy = read_json(json_path=cnvm_pkg_policy_data)
+    return agent_policy, package_policy
+
+
+if __name__ == "__main__":
+    # pylint: disable=duplicate-code
+    logger.info("Starting installation of CNVM AWS integration.")
+    agent_data, package_data = load_data()
+
+    logger.info("Create agent policy")
+    agent_policy_id = create_agent_policy(cfg=cnfg.elk_config, json_policy=agent_data)
+
+    logger.info("Create CNVM integration for policy", agent_policy_id)
+    package_policy_id = create_cnvm_integration(
+        cfg=cnfg.elk_config,
+        pkg_policy=package_data,
+        agent_policy_id=agent_policy_id,
+    )
+
+    save_state(
+        cnfg.state_data_file,
+        [
+            {
+                "pkg_policy_id": package_policy_id,
+                "agnt_policy_id": agent_policy_id,
+            },
+        ],
+    )
+    cloudformation_params = Munch()
+    cloudformation_params.ENROLLMENT_TOKEN = get_enrollment_token(
+        cfg=cnfg.elk_config,
+        policy_id=agent_policy_id,
+    )
+
+    cloudformation_params.STACK_NAME = "cnvm-sanity-test-stack"
+    cloudformation_params.FLEET_URL = get_fleet_server_host(cfg=cnfg.elk_config)
+    cloudformation_params.ELASTIC_AGENT_VERSION = get_agents(cfg=cnfg.elk_config)[0].agent.version
+    if "SNAPSHOT" in cloudformation_params.ELASTIC_AGENT_VERSION:
+        cloudformation_params.ELASTIC_ARTIFACTS_SERVER = (
+            cnfg.artifactory_url["snapshot"]
+            + get_build_info(
+                version=cloudformation_params.ELASTIC_AGENT_VERSION,
+                is_snapshot=True,
+            )
+            + CNVM_ARTIFACT_SUFFIX
+        )
+    else:
+        cloudformation_params.ELASTIC_ARTIFACTS_SERVER = (
+            cnfg.artifactory_url["staging"]
+            + get_build_info(
+                version=cloudformation_params.ELASTIC_AGENT_VERSION,
+                is_snapshot=False,
+            )
+            + CNVM_ARTIFACT_SUFFIX
+        )
```

Comment:
@amirbenun,  I’m not sure `ELASTIC_ARTIFACTS_SERVER` is correct
![Screenshot 2023-06-27 at 19 36 28](https://github.com/elastic/cloudbeat/assets/99176494/e45a9971-4c69-47a8-8b85-7a187f49aefc)